components/director/internal/model/api.go

```diff
@@ -43,32 +48,87 @@ func (_ *APIDefinition) GetType() resource.Type {
 }
 
 type APIDefinitionInput struct {
-	BundleID            *string
-	PackageID           *string
-	Tenant              string
-	Name                string
-	Description         *string
-	TargetURL           string
-	Group               *string //  group allows you to find the same API but in different version
-	OrdID               *string
-	ShortDescription    *string
-	SystemInstanceAware *bool
-	ApiProtocol         *string
-	Tags                json.RawMessage
-	Countries           json.RawMessage
-	Links               json.RawMessage
-	APIResourceLinks    json.RawMessage
-	ReleaseStatus       *string
-	SunsetDate          *string
-	Successor           *string
-	ChangeLogEntries    json.RawMessage
-	Labels              json.RawMessage
-	Visibility          *string
-	Disabled            *bool
-	PartOfProducts      json.RawMessage
-	LineOfBusiness      json.RawMessage
-	Industry            json.RawMessage
-	Version             *VersionInput
+	OrdBundleID         *string         `json:"partOfConsumptionBundle"`
+	OrdPackageID        *string         `json:"partOfPackage"`
+	Tenant              string          `json:",omitempty"`
+	Name                string          `json:"title"`
+	Description         *string         `json:"description"`
+	TargetURL           string          `json:"entryPoint"`
+	Group               *string         `json:",omitempty"` //  group allows you to find the same API but in different version
```

Comment:
These properties are relevant for the GraphQL API only and since there is a GraphQL layer that has relevant JSON tags these JSON tags on model entities are only relevant for the ORD scenario.